src/components/CookiesBanner/index.tsx

```diff
@@ -160,6 +160,13 @@ const CookiesBanner = (): ReactElement => {
     await saveCookie(COOKIES_KEY, newState, expDays)
     setShowAnalytics(localAnalytics)
     setShowIntercom(localIntercom)
+
+    // we remove GA cookies manually as react-ga does not provides a utility for it.
+    if (!localAnalytics) {
+      const subDomain = location.host.split('.').slice(-2).join('.')
+      COOKIES_LIST.forEach((cookie) => removeCookie(cookie.name, cookie.path, `.${subDomain}`))
+    }
```

Comment:
Can we extract this as an fn in src/utils/googleAnalytics.ts?

Reply:
good idea